common/views/components/WorkHeaderPrototype/WorkHeaderPrototype.tsx

```diff
@@ -104,7 +104,9 @@ const WorkHeaderPrototype: FunctionComponent<Props> = ({
             })}
             // We only send a lang if it's unambiguous -- better to send
             // no language than the wrong one.
-            lang={work.languages.length === 1 && work.languages[0].id}
+            lang={
+              work.languages.length === 1 ? work.languages[0].id : undefined
```

Comment:
`work.languages?.[0]?.id` ?

Reply:
cheers, ive changed this in all the areas with work.languages to work.languages[0]?.id